frontend/scripts/react-components/tool/tool.component.jsx

```diff
@@ -199,30 +209,41 @@ const Tool = ({ resizeSankeyTool }) => {
           <div className="c-modal js-modal" />
         </div>
 
-        {renderSankeyError()}
+        {renderSankeyError({ hasError, resetSankey })}
 
-        <div className="js-tool-content flow-content">
-          <div className="js-tool-loading tool-loading">
+        <div
+          className={cx('flow-content', {
+            '-center-map': isMapVisible,
+            open: isVisible
+          })}
+        >
+          <div className={cx('tool-loading', { 'is-visible': loading })}>
             <div className="veil sankey-veil" />
+            <Spinner />
             <div className="c-spinner" />
```

Comment:
it seems you have two spinners. I'm not sure this is the best way to show the spinner, can we include the veil in the component? Also might want to use the shrinking-spinner to use the same everywhere in the app. We might want to ask a designer about the spinner/veil situation.